src/main/webapp/room.js

```diff
@@ -17,30 +17,11 @@ const createUser = async (ev) => {
         },
       });
 
-    // Sends user to lobby.html on button click.
-    window.location.href = 'lobby.html';
-
-}
-
-const createRoom = async () => {
-  let room = {
-    roomId: Math.random().toString(36).substr(2, 9)
-  }
-
-  // Post room object
-  await fetch("/room", {
-    method: "POST",
-    body: JSON.stringify(room),
-    headers: {
-      "Content-Type": "application/json",
-    },
-  });
-  
+    // Sends user to newuser.html on button click.
+    window.location.href = 'newuser.html';
```

Comment:
Similar to what I said about "newuser.html", this file doesn't seem to be related to rooms anymore. Instead, it seems like it allows you to create a new user. So maybe this could be called "newuser.js" or just "user.js".

What do you think?

Reply:
@pruess-google sounds good, I changed it to user.js!